### PR TITLE
DevTools 105 > Application > Manifest > Protocol Handlers

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-105/#manifest

Also, replaced mentions of "Simulate Add to Homescreen events" with "Trigger installation" because [Chrome is streamlining PWA installation](https://developer.chrome.com/blog/a2hs-updates/) and the [absence of Home screen event link in DevTools confuses people](https://stackoverflow.com/questions/58128145/add-to-homescreen-link-missing-in-chrome-devtools).

Also, cosmetic changes.

Merge on Aug 30.

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/progressive-web-apps/index.md [hunk 1]
```diff
@@ -35,54 +40,69 @@ panel guides][2].
...
 mobile, too.
 
 If you want to test out the genuine mobile experience, you can connect a real mobile device to
-DevTools via \[remote debugging\][remote debugging][6], and then click the **Add to Homescreen**
-button (on DevTools) to trigger the "add to homescreen" prompt on the connected mobile device.
+DevTools via [remote debugging][6]. To trigger the installation on the connected mobile device, open the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Three-dot menu.", width="22", height="22" %} three-dot menu and click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/YwNSnnzuZ3dJwtVhcC4I.svg", alt="Install app.", width="24", height="24" %} **Install app**.
+
+### Test URL protocol handler registration {: #test-protocol-handler }
+
+The registration of such a handler lets installed PWAs handle links that use the URL protocol.
```
Comment (jecfish) [inline site/en/docs/devtools/progressive-web-apps/index.md:91]:
```suggestion
PWAs can handle links that use a specific protocol for a more integrated experience.
```

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/en/docs/devtools/progressive-web-apps/index.md [hunk 2]
```diff
@@ -35,54 +40,69 @@ panel guides][2].
...
+To learn how to create a handler, see [URL protocol handler registration for PWAs](https://web.dev/url-protocol-handler/).
+
+To test your handler:
+
+1. [Open DevTools](/docs/devtools/open/) on the landing page of your PWA. For example, check out this [demo page](https://protocol-handler.glitch.me/).
```
Comment (jecfish) [inline site/en/docs/devtools/progressive-web-apps/index.md:96]: Missing a step. To test the protocol handler (on successful navigation), one need to first install the app and reload and register.

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/en/docs/devtools/progressive-web-apps/index.md [hunk 3]
```diff
@@ -35,54 +40,70 @@ panel guides][2].
...
+
+### Test URL protocol handler registration {: #test-protocol-handler }
+
+PWAs can handle links that use a specific protocol for a more integrated experience.
+To learn how to create a handler, see [URL protocol handler registration for PWAs](https://web.dev/url-protocol-handler/).
+
+To test your handler:
+
+1. [Open DevTools](/docs/devtools/open/) on the landing page of your PWA. For example, check out this [demo page](https://protocol-handler.glitch.me/).
+1. From the demo page, install the PWA and reload the app after the installation. The browser has now registered the PWA as a handler for the `web+coffee` protocol with the operating system.
```
Comment (jecfish) [inline site/en/docs/devtools/progressive-web-apps/index.md:97]:
```suggestion
1. From the demo page, install the PWA and reload the app after the installation. The browser has now registered the PWA as a handler for the `web+coffee` protocol.
```

File: site/en/docs/devtools/progressive-web-apps/index.md [hunk 4]
```diff
@@ -35,54 +40,70 @@ panel guides][2].
...
+DevTools via [remote debugging][6]. To trigger the installation on the connected mobile device, open the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/N5Lkpdwpaz4YqRGFr2Ks.svg", alt="Three-dot menu.", width="22", height="22" %} three-dot menu and click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/YwNSnnzuZ3dJwtVhcC4I.svg", alt="Install app.", width="24", height="24" %} **Install app**.
+
+### Test URL protocol handler registration {: #test-protocol-handler }
+
+PWAs can handle links that use a specific protocol for a more integrated experience.
+To learn how to create a handler, see [URL protocol handler registration for PWAs](https://web.dev/url-protocol-handler/).
+
+To test your handler:
+
+1. [Open DevTools](/docs/devtools/open/) on the landing page of your PWA. For example, check out this [demo page](https://protocol-handler.glitch.me/).
```
Comment (jecfish) [inline site/en/docs/devtools/progressive-web-apps/index.md:96]:
```suggestion
1. [Open DevTools](/docs/devtools/open/) on the landing page of your PWA. For example, check out this [demo PWA](https://protocol-handler.glitch.me/).
```

**Review by jecfish — APPROVED** [review]
LGTM % minor comments.

**Pull request merged**